apps/src/storage/levelbuilder/ManifestEditor.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+
+class ManifestEditor extends React.Component {
+  static propTypes = {
+    // Provided via Redux
+    libraryManifest: PropTypes.object.isRequired
```

Comment:
If possible, can you change this to dataLibraryManifest or something similar? "library" is starting to become an overloaded term in our codebase.

Reply:
If you feel strongly about this I can do this as a separate PR. However, `libraryManifest` is part of the redux `DataState`, so I don't think it's likely to be confused with other libraries.

Reply:
Ok. Good call. This can be a separate PR (or part of a larger refactor to remove some of those other references to `library`